### PR TITLE
*: PoC to verify if unresolved conversations will block merging

### PR DESCRIPTION
**Pull request by s3nt3**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:

### Check List 

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
Please add a release note, or a 'None' if it is not needed.
```

### CODE REVIEW

File: poc02 [hunk 1]
```diff
@@ -0,0 +1 @@
+this_is_a_poc
```
Comment (s3nt3) [inline poc02:1]:
```suggestion
this_is_another_poc
```

Comment (wuhuizuo) [inline poc02:1]: should change?

**Review by wuhuizuo — APPROVED** [review]

**Pull request merged**